src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -16,8 +16,7 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
```

Comment:
I know it's the wrong line but would it be possible for you to change USERGUIDE_URL to point to our user guide?